module_utils/network_lsr/argument_validator.py

```diff
@@ -1822,6 +1835,10 @@ def __init__(self):
                 ArgValidatorDeprecated("master", deprecated_by="controller"),
                 ArgValidatorStr("interface_name", allow_empty=True),
                 ArgValidatorMac("mac"),
+                ArgValidatorMac(
+                    "cloned_mac",
```

Comment:
default_value="default" ?

Reply:
I removed it to work with default value, without it, yes I can define it here